websites/D/Doordash/presence.ts

```diff
@@ -0,0 +1,101 @@
+const presence = new Presence({
+		clientId: "1001831947611426877",
+	}),
+	started = Date.now();
```

Comment:
Update this at references too
```suggestion
	browsingTimestamp = Date.now();
```